docs/reference/migration/migrate_7_13.asciidoc

```diff
@@ -59,6 +59,27 @@ comma-delimited string. These are now passed one-by-one as single
 geohash strings.
 ====
 
+[discrete]
+[[breaking_713_ssl_changes]]
+==== SSL/TLS changes
+
+[[breaking_713_bundled_jdk_tls_versions]]
+.TLSv1.1 and TLSv1.0 are disabled in the bundled JDK
+[%collapsible]
+====
+*Details* +
+When using the bundled JDK, TLSv1.1 and TLSv1.0 are disabled by default.
```

Comment:
Is it helpful to link the JDK issue here?

Reply:
Yes. I mean to add it to 7.12 as well - thanks for the reminder.